types/types.go

```diff
@@ -114,6 +114,8 @@ const (
 	ControlPlaneName                 = "longhorn-manager"
 
 	DefaultRecurringJobConcurrency = 10
+	// DefaultRecurringJobTerminationGracePeriodSeconds set to 24 hours
```

Comment:
Why 24hrs?

Reply:
We should leave enough time for the backups to complete. I assume normally the backup shouldn't exceed beyond a day.